test/extensions/stats_sinks/wasm/BUILD

```diff
@@ -0,0 +1,25 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_test",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+envoy_cc_test(
+    name = "config_test",
+    srcs = ["config_test.cc"],
```

Comment:
Please rename this file to avoid misleading.

By convention, all extensions use `config_test.cc` to unit test `Factory` class (`WasmSinkFactory` in your case)